src/ExpressionEvaluator/Core/Source/ResultProvider/ResultProvider.cs

```diff
@@ -584,7 +622,7 @@ private void StoreResultAndContinue(DkmEvaluationResult result, ArrayBuilder<Eva
             }
         }
 
-        private void GetEvaluationResultsAndContinue(ArrayBuilder<EvalResultDataItem> rows, DkmEvaluationResult[] results, int index, int numRows, DkmWorkList workList, DkmInspectionContext inspectionContext, DkmStackWalkFrame stackFrame, CompletionRoutine completionRoutine)
+        private void GetEvaluationResultsAndContinue(ArrayBuilder<EvalResultDataItem> rows, DkmEvaluationResult[] results, int index, int numRows, WorkList workList, DkmInspectionContext inspectionContext, DkmStackWalkFrame stackFrame, CompletionRoutine completionRoutine)
         {
             if (index >= numRows)
```

Comment:
Will leave the check as is for now since we're calling this method from GetChildren and GetItems. But StoreResultAndContinue has been merged in and the condition reversed.